apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -127,11 +132,23 @@ const ScheduleModalContent = ({
         />
       </div>
 
+      {!isSubwayRoute(route) ? null : (
```

Comment:
The reason it isn't included, is the figma has upcoming departures after the daily schedule subway.
The non-subway has the daily schedule after the upcoming departures.
I would prefer them being in the same place in both version of the modals, but this is the design.
I also believe it is this way because the daily schedule for non-subway routes is much longer than the daily schedule for subway.

Reply:
Okay, fair enough!